src/main/java/org/iban4j/bban/BbanStructureEntry.java

```diff
@@ -104,12 +103,27 @@ public int getLength() {
     }
 
     public enum EntryCharacterType {
-        n,  // Digits (numeric characters 0 to 9 only)
-        a,  // Upper case letters (alphabetic characters A-Z only)
-        c  // upper and lower case alphanumeric characters (A-Z, a-z and 0-9)
+        /**
+         * Numerical digits (0-9 only)
+         */
+        n,
+        /**
+         * Alphabetical characters (A-Z only)
+         */
+        a,
+        // TODO 'c' used to say:  // upper and lower case alphanumeric characters (A-Z, a-z and 0-9)
+        //      but only generates A-Z0-9. Which is correct? Should it generate a-z too?
+        /**
+         * Combined alphabetical (uppercase) and numeric characters (A-Z and 0-9)
+         */
+        c
```

Comment:
hi @hajk1 can you help with this TODO please? I'm not sure what to do

Reply:
@aSemy sorry I missed this comment. You can keep the "A-Z0-9" which is more logical and also backward compatible.

Reply:
```suggestion
        /**
         * Combined alphabetical (uppercase) and numeric characters (A-Z and 0-9)
         */
        c
```